src/server/object_services/object_server.js

```diff
@@ -78,7 +78,11 @@ async function create_object_upload(req) {
         if (!req.rpc_params.etag) {
             throw new RpcError('INVALID_REQUEST', 'etag must be provided when using complete_upload');
         }
+        if (!req.rpc_params.last_modified_time) {
+            throw new RpcError('INVALID_REQUEST', 'last_modified_time must be provided when using complete_upload');
+        }
```

Comment:
I think this check can be relaxed - it is common for our object_mds to have just create_time and then we use it instead of modify_time.

Reply:
As discussed, I will keep this changes. We can always relax it in the future if there is need.